src/content/plugins.md

```diff
@@ -34,14 +30,19 @@ class MyPlugin extends Plugin {
 
 ## Constructor
 
-`inputNode`: This can be one of two things, a string or another Broccoli plugin. If a string is passed, Broccoli
-expects this to be a source directory and automatically converts this into a
-[broccoli-source](https://github.com/broccolijs/broccoli-source).
+```
+    constructor (inputNode, options) {
+        super([inputNodes], options);
```

Comment:
@oligriffiths this should probably be `inputNode` -> `[inputNode]` per Kevin's comment, correct?

Reply:
ah yes, oops